### ISSUE TITLE
Google Colab demo with video input

### ISSUE THREAD

**Issue by sharmaabhishekk** [YuliangXiu/PoseFlow#25]
Thank you for this repository. 

I am trying to get a AlphaPose+PoseFlow workflow on videos to run on Google Colab. The AlphaPose part is fine - I'm getting the `alphapose-results.json`. 

However, how do I run PoseFlow on it (since I don't have a image directory -just a video - and `image_dir` is a necessary argument for `tracker-general.py`)

My notebook is [here](https://gist.github.com/sharmaabhishekk/c8135ee767e16d6e488d068d82160294).

**Comment by jpbranson** [comment]
The only solution I could find was to [extract all the video frames](https://stackoverflow.com/questions/34786669/extract-all-video-frames-as-images-with-ffmpeg/34786929) into a directory - run AlphaPose on that directory, then use the result to run PoseFlow.

**Comment by sharmaabhishekk** [comment]
Yup, ended up doing that as well. I guess I'll leave this open for now in case the authors have a better solution to this.

**Comment by FCrSTATS** [comment]
I just changed xrange to range in the general_tracker.py file and it seemed to work in python, but maybe it impacted the results. Would this fix provide clean results?

**Comment by jpbranson** [comment]
@FCrSTATS It would make sense to provide the same results. We could try running it on the same clip to test?

**Comment by neeek2303** [comment]
@sharmaabhishekk hi! Could you please share your code with PoseFlow part?
